packages/messaging/lib/sendMessageWrapper.js

```diff
@@ -0,0 +1,144 @@
+import { sendMessage } from '../../../src/utils'
+
+export class SendMessageMessagingConfig {}
+
+/**
+ * An temporary implementation of {@link MessagingTransport} to communicate with Android and Extension.
+ * It wraps the current messaging system that calls `sendMessage`
+ *
+ * @implements {MessagingTransport}
+ */
+export class SendMessageMessagingTransport {
```

Comment:
This entire construct can be moved into the CTL folder and we keep it very specific to the migration period of extensions/android.